### DISCUSSION TITLE
Set <Down>, <Up> to jump in snip from custom settings

### DISCUSSION THREAD

**Discussion by datfooldive** [NvChad/NvChad#2605] [Q&A] [answered]
Hi, i need little help how to set `<Down>`  to next snip or `<Up>` for prev and make `<Tab>` as expand from custom settings. i try to copy `plugins.configs.cmp` `<Tab>`, `<S-Tab>` config to `custom.mapping` but cmp not find. sorry for bad english :)

**Comment by siduck** [comment, marked as answer]
@kyzsuukii never override custom mappings to add cmp ones, cuz cmp mappings are in its config itself. so u better override its config

look at the last example here, hint for u!

https://nvchad.com/docs/config/plugins
